### ISSUE TITLE
Custom Pricing Not Working on Product View

### ISSUE THREAD

**Issue by ghost** [justbetter/magento2-customer-pricing#3]
Custom pricing is working on the list view for products, but not on the actual product page.

List View Product Pricing:
<img width="227" alt="image" src="https://github.com/justbetter/magento2-customer-pricing/assets/60128608/4e469b2f-456d-487f-9a87-4308e2ca423a">

Product View Pricing:
<img width="356" alt="image" src="https://github.com/justbetter/magento2-customer-pricing/assets/60128608/f9e58f7b-11be-4c9c-aa43-ba7f94314dae">

This is using the base Magento 2 Luma Theme.

If you hook into the `catalog_product_load_after` event it works.

```
<?xml version="1.0"?>
<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Event/etc/events.xsd">
    <event name="catalog_product_get_final_price">
        <observer name="customer_specific_pricing" instance="JustBetter\CustomerPricing\Observer\FinalPrice" />
    </event>
    <event name="catalog_product_collection_load_after">
        <observer name="custom_price_override_category" instance="JustBetter\CustomerPricing\Observer\CategoryPricing" />
    </event>
    <event name="catalog_product_load_after">
        <observer name="customer_specific_pricing" instance="JustBetter\CustomerPricing\Observer\FinalPrice" />
    </event>
</config>
```

**Comment by VincentBean** [comment]
Thank you for your feedback, the final price observer should fix this but if it doesn't could you make a PR to fix this?

**Comment by VincentBean** [comment]
Fixed by #4
